pkg/controller/certificates/sync.go

```diff
@@ -144,7 +147,16 @@ func (c *Controller) Sync(ctx context.Context, crt *v1alpha1.Certificate) (err e
 	// listed in the certificate spec, we should re-issue the certificate.
 	if k8sErrors.IsNotFound(err) || errors.IsInvalidData(err) ||
 		expectedCN != cert.Subject.CommonName || !util.EqualUnsorted(cert.DNSNames, expectedDNSNames) {
-		return c.issue(ctx, i, crtCopy)
+
+		err := c.issue(ctx, i, crtCopy)
+
+		c.metrics.HandleCertificateIssue(crt, issuerObj, err)
```

Comment:
For ACME certificates, this area of code could be called >1 right now for a single certificate issuance.

Reply:
I'm not sure how to handle that, any recommendations?